lib/cc/analyzer/container.rb

```diff
@@ -158,8 +164,13 @@ def container_data(duration: nil, status: nil)
         ContainerData.new(@image, @name, duration, status, @stderr_io.string)
       end
 
-      def reap_running_container
-        Analyzer.logger.warn("killing container name=#{@name}")
+      def kill_reader_threads
+        @t_out.kill if @t_out
+        @t_err.kill if @t_err
+      end
+
+      def reap_running_container(message)
+        Analyzer.logger.warn("killing container name=#{@name} message=#{message.inspect}")
         POSIX::Spawn::Child.new("docker", "kill", @name)
```

Comment:
[`docker stop`](https://docs.docker.com/engine/reference/commandline/stop/) will signal `SIGTERM` and then kill after a time. Would there be any benefit in using that here?